extensions/reviewed/CrazyGamesAdApi.json

```
@@ -561,6 +627,291 @@
+      "name": "DisplayResponsiveBanner",
+      "private": true,
+      "sentence": "Display a responsive banner: _PARAM1_",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::Standard",
+          "conditions": [
+            {
+              "type": {
+                "inverted": true,
+                "value": "CrazyGamesAdApi::IsSDKLoaded"
+              },
+              "parameters": [
+                "",
+                ""
+              ]
+            }
+          ],
+          "actions": [
+            {
+              "type": {
+                "value": "BuiltinAsync::ResolveAsyncEventsFunction"
+              },
+              "parameters": [
+                ""
+              ]
+            }
+          ]
+        },
+        {
+          "type": "BuiltinCommonInstructions::Standard",
+          "conditions": [
+            {
+              "type": {
+                "value": "CrazyGamesAdApi::IsSDKLoaded"
+              },
+              "parameters": [
+                "",
+                ""
+              ]
+            }
+          ],
+          "actions": [],
+          "events": [
+            {
+              "type": "BuiltinCommonInstructions::JsCode",
+              "inlineCode": [
+                "const logger = new gdjs.Logger(\"CrazyGames SDK\");",
+                "const bannerName = eventsFunctionContext.getArgument(\"BannerName\");",
+                "",
+                "//Create a container for the responsive banner if needed",
+                "if (document.getElementById(bannerName) === null) {",
+                "    const div = document.createElement('div');",
+                "    div.id = bannerName;",
+                "    div.setAttribute(\"style\", \"pointer-events: none; position:fixed; z-index:1; width: 300px; height: 250px;\");",
+                "",
+                "    const body = document.getElementsByTagName(\"body\")[0];",
+                "    // Insert the new container in first position in the body",
+                "    body.insertBefore(div, body.firstChild);",
+                "}",
+                "",
+                "window.CrazyGames.SDK.banner.requestResponsiveBanner(bannerName, (error, result) => {",
+                "    if (error) {",
+                "        logger.error(\"Error on request responsive banner.\", error);",
+                "    } else {",
+                "        // result is always undefined when requesting banners",
+                "        logger.log(\"End request responsive banner\", result === undefined ? \"Requesting banners\" : result);",
+                "        eventsFunctionContext.task.resolve();",
+                "    }",
+                "});"
+              ],
+              "parameterObjects": "",
+              "useStrict": true,
+              "eventsSheetExpanded": true
+            }
+          ]
+        }
+      ],
+      "parameters": [
+        {
+          "description": "Banner's name",
```

A banner is not a person.
```suggestion
          "description": "Banner name",
```